vizro-core/examples/visual-vocabulary/assets/css/custom.css

```diff
@@ -73,10 +73,6 @@ img[src*="#chart-icon"] {
   margin: 0;
 }
 
-.intro-text p a {
```

Comment:
@lingyielia @nadijagraca or @maxschulz-COL  - this can be removed from the vizro-ai custom.css as well after this merge. I think you've guys used it somewhere?